<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementListWidget</class>
 <widget class="QWidget" name="ElementListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="treeView_elements"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btn_addlayer">
       <property name="text">
        <string>add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_removelayer">
       <property name="text">
        <string>remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_moveup">
       <property name="text">
        <string>up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_movedown">
       <property name="text">
        <string>down</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
